packages/one-app-server-bundler/__tests__/utils/getConfigOptions.spec.js

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2019 American Express Travel Related Services Company, Inc.
```

Comment:
I don't know if these need to be updated as they are technically new files.

Reply:
Addressed